GithubPractice/ViewController.swift

```diff
@@ -30,25 +32,37 @@ class ViewController: UIViewController {
         lastNameField.delegate = self
         jobPositionField.delegate = self
         
+        profileImage.layer.cornerRadius = profileImage.frame.height/2
+        
+        if let fisrtName = defaults.string(forKey: firstNameKey) {
```

Comment:
like here Constants.firstNameKey insead of firstNameKey

Reply:
I thought the same, but then I thought that maybe the code is clearer in this way